arcgis-ios-sdk-samples/AppDelegate.swift

```diff
@@ -174,19 +174,22 @@ extension UIColor {
 extension UIApplication {
     /// License the app with ArcGIS Runtime deployment license keys.
     ///
-    /// - Note: An invalid key does not throw an exception, but simply fails to license the app,
-    ///         falling back to Developer Mode (which will display a watermark on the map view).
+    /// - Note: An invalid key does not throw an exception, but simply fails to
+    ///         license the app, falling back to Developer Mode (which will
+    ///         display a watermark on the map view).
     func license() {
         do {
-            // Don't set extension key if you don't use utility network samples.
-            // try AGSArcGISRuntimeEnvironment.setLicenseKey(.licenseKey)
-            
-            // Set both keys for accessing all samples.
-            try AGSArcGISRuntimeEnvironment.setLicenseKey(.licenseKey, extensions: [.extensionLicenseKey])
+            if let licenseKey = String.licenseKey,
+               let extensionLicenseKey = String.extensionLicenseKey {
+                // Set both keys to access all samples, including utility network capability.
+                try AGSArcGISRuntimeEnvironment.setLicenseKey(licenseKey, extensions: [extensionLicenseKey])
+            }
             
-            // Authentication with an API key or named user is required to
-            // access basemaps and other location services.
-            AGSArcGISRuntimeEnvironment.apiKey = .apiKey
+            if let apiKey = String.apiKey {
+                // Authentication with an API key or named user is required to
+                // access basemaps and other location services.
+                AGSArcGISRuntimeEnvironment.apiKey = apiKey
+            }
```

Comment:
This code doesn't throw, so I don't think it makes sense to have it within the `do...catch` block. In fact, the `do...catch` block should probably be inside the first `if` statement.